src/index.ts

```diff
@@ -242,6 +243,16 @@ function getConfig(userConfig?: Partial<Config>): Config {
     const result = getEnvValue(mergeXrayTracesEnvVar, "false").toLowerCase();
     config.mergeDatadogXrayTraces = result === "true";
   }
+  if (userConfig === undefined || userConfig.traceExtractor === undefined) {
+    try {
+      const extractorEnv = getEnvValue(traceExtractorEnvVar, "");
+      const taskRootEnv = getEnvValue(lambdaTaskRootEnvVar, "");
```

Comment:
Could this cause a circular import issues? Say my-handler.js -> datadog-lambda-js -> my-handler.js? Also, we shouldn't call require if either value is unset.

Reply:
Moved it to the handler file! So it can be injected as a config too Please review!